colonel/simulator/simulator.py

```diff
@@ -44,20 +49,35 @@ def parse_output_file(cls, file_path):
         return pd.read_csv(file_path, delimiter=r'\s+',
                            names=[cls.TIME_COL, cls.PORT_COL, cls.VALUE_COL])
 
+    @staticmethod
+    def _is_list_value(value: str) -> bool:
+        return '[' in value and ']' in value
```

Comment:
Maybe it would be more secure to do sth like:
```
return value.strip().startsWith("[") and value.strip().endsWith("]")
```

Reply:
yeah, that seems better